firebase-firestore/ktx/src/androidTest/java/com/google/firebase/firestore/FirestoreMapEncoderIntegrationTest.kt

```diff
@@ -90,10 +90,11 @@ class FirestoreMapEncoderIntegrationTest {
             )
 
         for (student in studentList) {
-            assertFailsWith<IllegalArgumentException>(
-                message = "not supported, please use",
-                block = { docRefKotlin.setData(student) }
-            )
+            assertThrows<IllegalArgumentException> {
```

Comment:
Why the change?

Reply:
As you suggested before, I am using a high order test util function to use `Truth` under the hood to do this assertThrow check, so this makes it easy for me to debug when it fails. 

The original one uses the android test JUnit library, I have a TODO in my ktx.gradle file to remove this dependency after I merge this decoder PR (because this Truth assertThrow helpper method is only in this decoder PR so far).